app/src/main/java/com/mailerdaemon/app/placement/PlacementActivity.kt

```diff
@@ -102,8 +102,8 @@ class PlacementActivity : AppCompatActivity(), AccessDatabase {
                 if (newText == "") {
                     updateRV(data ?: emptyList())
                 } else {
-                    val list = data?.filter {
-                        it.message.contains(newText!!, true)
+                    val list = data.filter {
+                        it.message.contains(requireNotNull(newText), true)
```

Comment:
requireNonNull does the same thing as `!!`. instead you can change If condition to `If(newText==" " || newText==null)`